### ISSUE TITLE
Missing close button

### ISSUE THREAD

**Issue by mlewand** [cksource/ckeditor-plugin-a11ychecker#159]
Turns out that down the road somehow we lost close button... :)

We need to make sure that there are tests for that regression.

![image](https://cloud.githubusercontent.com/assets/5353898/13004196/588fa274-d17a-11e5-9165-9c74941aebd9.png)
